README.md

```diff
@@ -1,2 +1,94 @@
 # utPLSQL-maven-plugin
 A maven plugin for running Unit Tests with utPLSQL v3+
+
+
+### Configuration
+
+* `url`
+  * URL of the Connection to the database
+  * Default: `jdbc:oracle:thin:@localhost:1521:ut3`
+* `user`
+  * Credential of the connection to the database
+  * Default: `ut3`
+* `password`
+  * Password of the connection to the database
+  * Default: `XNtxj8eEgA6X6b6f`
+* `ignoreFailure`
+  * Ignore or continue when a test fail
+  * Default: `${maven.test.failure.ignore}`
+* `colorConsole`
```

Comment:
We could get aligned with maven settings and not request users to set this flag explicitly.
https://stackoverflow.com/questions/43638595/how-do-i-remove-colors-from-maven-output

Reply:
I just updated the configuration in order to use the maven standard mechanism but I don't have time to test it.